src/Microsoft.AspNetCore.Http.Features/Authentication/IHttpAuthenticationFeature.cs

```diff
@@ -9,6 +10,7 @@ public interface IHttpAuthenticationFeature
     {
         ClaimsPrincipal User { get; set; }
 
+        [Obsolete("This is obsolete and will be removed in a future version. The recommended alternative is to use Microsoft.AspNetCore.Authentication.AuthenticationHttpContextExtensions. See https://go.microsoft.com/fwlink/?linkid=845470.")]
```

Comment:
Drop "The recommended alternative is to use Microsoft.AspNetCore.Authentication.AuthenticationHttpContextExtensions."? It's not a direct replacement for this API.